src/main/java/Duke.java

```diff
@@ -1,10 +1,103 @@
+
+import java.util.Scanner;
+
 public class Duke {
     public static void main(String[] args) {
```

Comment:
One of the coding standards is to avoid long methods, maybe you can consider refactoring the code to have a method for analysing inputs, and then from there call the respective functions to carry out the necessary actions